sdk/attestation/Azure.Security.Attestation/src/AttestationClient.cs

```diff
@@ -338,7 +338,7 @@ public virtual async Task<Response<BinaryData>> AttestTpmAsync(BinaryData reques
         /// Retrieves the signing certificates used to sign attestation requests.
```

Comment:
Apologies, I can't suggest directly there, but for L335, it may be helpful to provide the warning context for readers:

`#pragma warning restore CA1822  // Mark members as static`